flink/src/test/java/org/apache/iceberg/flink/actions/TestRewriteDataFilesAction.java

```diff
@@ -280,15 +280,15 @@ public void testRewriteLargeTableHasResiduals() throws IOException {
       Assert.assertEquals("Residuals must be ignored", Expressions.alwaysTrue(), task.residual());
     }
     List<DataFile> dataFiles = Lists.newArrayList(CloseableIterable.transform(tasks, FileScanTask::file));
-    Assert.assertEquals("Should have 2 data files before rewrite", 2, dataFiles.size());
```

Comment:
the default parallelism is 4 ,we have 2 `insert into` ,so it will generate 8 datafiles

Reply:
Where did we have change the default parallelism from 1 to 4 ?  I did not find the changes .

Reply:
It is [here](https://github.com/apache/iceberg/blob/master/flink/src/test/java/org/apache/iceberg/flink/MiniClusterResource.java#L30)

Reply:
when I debug in flink 1.12 source code ,it seem that use this value  to  override the  default parallelism

Reply:
That looks strange because when we tested this patch (https://github.com/apache/iceberg/commit/c5e67915b01c2cbf2eea40a412c2765f0c8278e8) in travis ci, it did not  report the parallelism issue, but in this DynamicTable refactor issue, the parallelism issue happen.  That don't make sense for me.

Reply:
Here,  I think we'd better to set the parallelism to the default value 1.

Reply:
I updated it